src/app/csc/csc.component.ts

```diff
@@ -283,12 +283,13 @@ export class CscComponent implements OnInit {
         if (
           this.cscProjectsList[project].contacts.principal_investigators != null
         ) {
-          this.cscProjectsList[project].investigators_formatted = "";
+          this.cscProjectsList[project].investigators_formatted = "<ul>";
```

Comment:
Just to reduce change this list should be styled without a bullet

Reply:
Done